src/components/Announcements/AnnouncementsDialog.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, FC } from 'react'
+import SwipeableViews from 'react-swipeable-views'
+
+import { FixedActionsDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import CozyTheme from 'cozy-ui/transpiled/react/providers/CozyTheme'
+import MobileStepper from 'cozy-ui/transpiled/react/MobileStepper'
+import IconButton from 'cozy-ui/transpiled/react/IconButton'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import LeftIcon from 'cozy-ui/transpiled/react/Icons/Left'
+import RightIcon from 'cozy-ui/transpiled/react/Icons/Right'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+
+import { AnnouncementsDialogContent } from './AnnouncementsDialogContent'
+import { Announcement } from './types'
+import { useAnnouncementsSettings } from 'hooks/useAnnouncementsSettings'
+
+interface AnnouncementsDialogProps {
+  announcements: Array<Announcement>
+  onDismiss: () => void
+}
+
+const AnnouncementsDialog: FC<AnnouncementsDialogProps> = ({
+  announcements,
+  onDismiss
+}) => {
+  const { values, save } = useAnnouncementsSettings()
+  const { isMobile } = useBreakpoints()
+
+  const [activeStep, setActiveStep] = useState(0)
+
+  const handleBack = (): void => {
+    setActiveStep(activeStep - 1)
+  }
+
+  const handleNext = (): void => {
+    const uuid = announcements[activeStep].attributes.uuid
+    if (!values?.seen.includes(uuid)) {
+      save({
+        seen: [...(values?.seen ?? []), uuid]
+      })
+    }
```

Comment:
do we want to mark a announcement as read when clicking `back` too?

Reply:
No, we always start with the first news item. As a result, users don't see any new news unless they click Next